src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -70,8 +80,9 @@ else if (elevatorValue > Constants.Swerve.maxElevatorValue && speed > 0){
         SmartDashboard.putNumber("Encoder Value", rightElevator.getEncoder().getPosition());
 
     }
-    public void spinKicker(double percent){
-        kicker.set(ControlMode.PercentOutput, percent);
+    public void setFlyWheels(double percent){
+        // kicker.set(ControlMode.PercentOutput, percent);
```

Comment:
We should remove this commented out line as well.

Reply:
removed